ajax/libs/print.js/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "print.js",
+  "filename": "print.min.js",
+  "homepage": "http://printjs.crabbly.com",
+  "author": "Rodrigo Vieira <rodrigo@crabbly.com>",
+  "description": "A tiny javascript library to help printing from the web.",
+  "version": "1.0.24",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/crabbly/Print.js.git"
+  },
+  "keywords": [
+    "javascript",
+    "print.js",
+    "printjs"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/crabbly/Print.js.git",
+    "fileMap": [
```

Comment:
How about using npm auto-update?
the files of [v1.0.28](https://github.com/crabbly/Print.js/tree/v1.0.28) can be used in front-end from npm, but there is no `dist` folder on git repo.
Or maybe let's discuss this issue with the author, what do you think?
Many thanks.

Reply:
Hey guys, I accepted a PR recently that removed the `dist` folder from commits. I could add it back if this can get the library in cdnjs.
Other than that, is there anything else I need to do?
Let's get this done.
Thank you.

Reply:
@crabbly That will be great if you will still use dist folder, Thanks for you cooperation

Reply:
@sufuf3 There is a homepage in config
Also, the author will add dist folder to latest version and (I hope) will use dist folder further

Reply:
@sashberd @sufuf3 The `dist` folder has been added back with the latest release.
https://github.com/crabbly/Print.js/releases/latest
It is also already published to npm. 
Thank you.